### PR TITLE
updated races

### PR DESCRIPTION
**Pull request by sethb4**
Contributes to #76 in Open-Sentencing

## What did you do?
Updated the race options for clients.

## Why did you do it?
Requested by the UI team.

## How have you tested it?
Some tests were run through the endpoints.

## Were docs updated if needed?

- [ ] No
- [ ] Yes
- [x] N/A

#### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new console logs
- [ ] Fixes entire issue
- [x] Partial fix for issue

### CODE REVIEW

File: src/main/java/io/swagger/model/Client.java [hunk 1]
```diff
@@ -39,25 +38,13 @@
...
-    BIRACIAL("Biracial"),
-    
-    WHITE_HISPANIC_OR_LATINO_("White [Hispanic or Latino]"),
-    
-    WHITE_BLACK_HISPANIC_OR_LATINO_("White/Black [Hispanic or Latino]"),
-    
-    UNKNOWN("Unknown");
+    BLACK_OR_AFRICAN_AMERICAN("Black or African American"),
+    HISPANIC_OR_LATINO("Hispanic or Latino"),
+    NATIVE_HAWAIIN_OR_OTHER_PACIFIC_ISLANDER("Native Hawaiian or Other Pacific Islander"),
```
Comment (edelgrace) [inline src/main/java/io/swagger/model/Client.java:45]: Hawaiian is spelled wrong.

HAWAIIN -> HAWAIIAN

**Review by edelgrace — COMMENTED** [review]
Just need to fix a typo (HAWAIIN -> HAWAIIAN) and then I think it looks good!

File: src/main/java/io/swagger/model/Client.java (hunk 1 above)
Comment (sethb4) [reply src/main/java/io/swagger/model/Client.java]: Good catch, this has been fixed.

**Review by upkarlidder — APPROVED** [review]
Discussed with @demilolu.

**Pull request merged**